python/setuptools_entry_points.md

```diff
@@ -0,0 +1,61 @@
+# Setuptools entry points
+The most common way of exposing a Python CLI is to use the "scripts"
+keyword in ``setup.py``.
```

Comment:
That's true - I could develop that sentence a bit. Would something as simple as this work?

> When building a Python package, the most common way of exposing a command line interface (CLI)...